RecoEgamma/EgammaIsolationAlgos/python/gamIsoFromDeposits_cff.py

```diff
@@ -2,11 +2,11 @@
 
 from RecoEgamma.EgammaIsolationAlgos.gamIsoFromDepsModules_cff import *
 
-gamIsoFromDeposits = cms.Sequence( 
-    gamIsoFromDepsTk *
-    gamIsoFromDepsEcalFromHitsByCrystal * 
-    gamIsoFromDepsHcalFromTowers *
-    gamIsoFromDepsHcalDepth1FromTowers *
+gamIsoFromDepositsTask = cms.Task( 
+    gamIsoFromDepsTk ,
+    gamIsoFromDepsEcalFromHitsByCrystal , 
+    gamIsoFromDepsHcalFromTowers ,
+    gamIsoFromDepsHcalDepth1FromTowers ,
     gamIsoFromDepsHcalDepth2FromTowers
 )
-
+gamIsoFromDeposits = cms.Sequence(gamIsoFromDepositsTask) 
```

Comment:
This is only used in ```RecoEgamma/EgammaIsolationAlgos/python/gamIsolationSequence_cff.py```: maybe one could avoid defining explicitly the sequence here